cypress/integration/visitorCanViewListOfArticles.spec.js

```diff
@@ -0,0 +1,53 @@
+describe('Visitor can view on the landing page', () => {
+
+  it('a list of articles successfully', () => {
+    cy.server();
+    cy.route({
+      method: 'GET',
+      url: 'http://localhost:3000/api/v1/articles',
+      response: 'fixture:list_of_articles.json',
+      status: 200
+    })
+    cy.visit('http://localhost:3001')
+    cy.get('#id_1').within(() => {
```

Comment:
You can simplify this by using standard javascript looping function such as `forEach()`
For example:
```
 let form = [
      ["input[name=email]", "coach@mail.com"],
      ["input[name=password]", "password"],
      ["input[name=password_confirmation]", "password"],
      ["input[name=first_name]", "Jack"],
      ["input[name=last_name]", "Black"]
    ]
    
    form.forEach(element => {
      cy.get(element[0]).type(element[1]);
    });
```
For reference :) 
https://medium.com/craft-academy/5-awesome-tricks-to-turbocharge-your-react-automated-tests-147776289c55

Reply:
done